openmdao/core/system.py

```diff
@@ -2605,15 +2618,6 @@ def _solve_linear(self, vec_names, mode):
             list of names of the right-hand-side vectors.
         mode : str
             'fwd' or 'rev'.
-
```

Comment:
This method is always overriden, and the versions of it in subclasses have these returns. Don't we want the parent class stubs to have the fully documented docstring args and returns? I don't know the answer, but what does eveyone think.